lib/manager/flux/artifacts.ts

```diff
@@ -0,0 +1,50 @@
+import { logger } from '../../logger';
+import { exec } from '../../util/exec';
+import type { ExecOptions } from '../../util/exec/types';
+import type { UpdateArtifact, UpdateArtifactsResult } from '../types';
+import { isSystemManifest } from '.';
+
+export async function updateArtifacts({
+  packageFileName,
+  updatedDeps,
+}: UpdateArtifact): Promise<UpdateArtifactsResult[] | null> {
+  if (!isSystemManifest(packageFileName) || !updatedDeps[0]?.newVersion) {
+    return null;
+  }
+  try {
+    logger.debug(`Updating Flux system manifests`);
+    const cmd = 'flux install --export';
+    const execOptions: ExecOptions = {
+      docker: {
+        image: 'sidecar',
+      },
+      toolConstraints: [
+        {
+          toolName: 'flux',
+          constraint: updatedDeps[0].newVersion,
+        },
+      ],
+    };
+    const result = await exec(cmd, execOptions);
+
+    return [
+      {
+        file: {
+          type: 'addition',
+          path: packageFileName,
```

Comment:
The only file which changes the manifest file itself?

Reply:
That's correct.